iocore/net/UnixNetAccept.cc

```diff
@@ -277,14 +277,12 @@ NetAccept::do_blocking_accept(EThread *t)
     vc->options.packet_mark = packet_mark;
     vc->options.packet_tos = packet_tos;
     vc->apply_options();
-    vc->from_accept_thread = true;
```

Comment:
Should this line be removed within this change?
I'm fine with introducing the context, however, I guess it is not a part of it. This makes me nervous.